test/library/packages/LinearAlgebra/correctness/testPowSparse.chpl

```diff
@@ -0,0 +1,20 @@
+use LinearAlgebra;
```

Comment:
I have generally been trying to keep most of the correctness coverage tests in `correctness.chpl` to make it easier to identify if a feature has a test and to reduce overall compilation time. I'm fine with this test being a standalone test, but it may get merged into another file at some point.

Reply:
Do you want me to merge this into `correctness.chpl`?

Reply:
If it's not a hassle. It may make sense to break `correctness.chpl` up into subsections or at least the submodules some day, but I don't expect it to ever reach the granularity of individual functions.